### DISCUSSION TITLE
Is there an easier way to forcefully kill a window?

### DISCUSSION THREAD

**Discussion by oriongonza** [i3/i3#5846] [Q&A]
[The documentation](https://i3wm.org/docs/userguide.html#_closing_windows) allows for this, but sometimes process get stuck or are annoying to shutdown. 
I've written this convoluted command, is there something better?
`bindsym $mod+shift+q exec "xprop -id $(xdotool getwindowfocus) _NET_WM_PID | awk '{print $3}' | xargs kill"`

**Comment by ronggothelast** [comment]
try "xkill" and select the window
